src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/OveruseEstimator.java

```diff
@@ -86,7 +86,8 @@
 
     private double slope;
 
-    private final List<Double> tsDeltaHist = new LinkedList<>();
+    private final double[] tsDeltaHist = new double[kMinFramePeriodHistoryLength];
+    private int tsDeltaHistIdx = 0;
```

Comment:
As you've probably figured out based on the sparse class comment that I've put, OveruseDetector was translated from C++. Consequently, it lacks javadocs where there no equivalents in the respective C++ source code.

However, we need comments where we author source code. And in your case, I'd say that javadocs are pretty necessary because you're practically implementing a circular buffer and that's complex enough to not be immediately recognizable. Without good javadocs, I'd say the supposed performance improvement is not big enough to justify modifying a translated piece.